dev-docs/show-video-with-a-dfp-video-tag.md

```diff
@@ -0,0 +1,174 @@
+necessary, but was helpful during development.  Specifically, we log:
+
++ the `bids` object we got back from our demand sources
++ a notice telling us whether there was video demand
++ the URL of the VAST creative, if any (this is also helpful if you
+  don't necessarily use DFP)
+
+All of the code in this step and step # 1 should be in the page header.
+
+```javascript
+pbjs.que.push(function(){
+  pbjs.addAdUnits(videoAdUnit);
+
+  pbjs.requestBids({
+    timeout : 700,
+    bidsBackHandler : function(bids) {
+      console.log('got bids back: ');
+      console.log(bids);
+      if (!bids.video) {
+        console.log('no video demand');
+      }
+      else {
+        console.log('we got video demand!');
+      }
+
+      // Log the VAST URL, if there is one
+      try {
+        url = bids.video.bids[0].vastUrl;
+        console.log('VAST URL: ');
+        console.log(url);
+      } catch (e) {} // ignore
+
+      // This is the example tag from https://support.google.com/dfp_premium/answer/1068325
+      var adserverTag = 'http://pubads.g.doubleclick.net/gampad/ads?env=vp&gdfp_req=1&impl=s&output=vast&iu=/6062/video-demo&sz=400x300&unviewed_position_start=1&url=http://www.simplevideoad.com&ciu_szs=728x90,300x250&correlator=7105';
+
+      var options = {
+        'adserver': 'dfp',
+        'code': 'video' // Must match the code from the `videoAdUnit` above
+      };
+
+      // Generate DFP Video Ad Server Tag URL
+      var masterTagUrl = pbjs.buildMasterVideoTagFromAdserverTag(adserverTag, options);
+
+      console.log('buildMasterVideoTagFromAdserverTag: ' + masterTagUrl);
+
+      // Send masterTagUrl to the video player
+      invokeVideoPlayer(masterTagUrl);
+    }
+  });
+});
+```
+
+### 3. Add the video player code to the page body
+
+In the body of the page, some HTML and JS like the following will show
+the ad -- this is where `invokeVideoPlayer` is defined:
+
+```html
+<div class="example-video-container">
+  <video id="vid1" class="video-js vjs-default-skin vjs-big-play-centered" controls 
+    data-setup='{}'
+    width='640'
+    height='480'
+  >
+    <source src="http://vjs.zencdn.net/v/oceans.mp4" type='video/mp4'/>
+    <source src="http://vjs.zencdn.net/v/oceans.webm" type='video/webm'/>
+    <source src="http://vjs.zencdn.net/v/oceans.ogv" type='video/ogg'/>    
+  </video>
+</div>
+
+<script>
+
+var vid1 = videojs('vid1');
+
+function invokeVideoPlayer(url) {
+  videojs("vid1").ready(function() {
+    var player = this;
+    var vastAd = player.vastClient({
+      adTagUrl: url,
+      playAdAlways: true,
+      vpaidFlashLoaderPath: "https://github.com/MailOnline/videojs-vast-vpaid/blob/RELEASE/bin/VPAIDFlash.swf?raw=true",
+      autoplay: true
+    });
+    player.muted(true);
+    player.play();
+  });
+}
+
+</script>
+```
+
+At this point you should be able to see an instream preroll video ad,
+followed by the ocean video from the
+[Video.js homepage](http://videojs.com/).
+
+</div>
```

Comment:
Looks good, I would add a link somewhere on this page that references the `setting-up-prebid-video-in-dfp.html` page.